config.php

```diff
@@ -81,7 +81,7 @@
 define(PHP_SELF, 'imgboard.php');						//Name of main script file
 define(PHP_SELF2, 'index.html');						//Name of main htm file
 define(PHP_EXT, '.html');								//Extension used for board pages after first
-define(JS_PATH, 'jquery'); 								//relative path from imgboard.php of the jquery folder without a trailing slash
+define(JS_PATH, 'plugins/jquery'); 								//relative path from imgboard.php of the jquery folder without a trailing slash
```

Comment:
Whatever's easiest and does the least damage, I didn't want to mess with the suite